package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "liquibase",
-  "version": "2.0.5",
+  "name": "@rimac-automobili/liquibase",
```

Comment:
This probably shouldn't go in the PR, same for version bump

Reply:
@vuki656  @ijakab yeah we'll want to remove that before merging